crm_deduplicate_by_fuzzy_name/wizards/partner_merge.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import osv
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning as UserError
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    # The field is named this way to not be automatically added as a "group by"
+    # option, which is performed in the code through the name of the field
+    fuzzy_name_group_by = fields.Boolean(
+        string='Similar name',
+        help="WARNING: This option is significantly slower than the others, "
+             "because the match cannot be performed in group.")
+    max_fuzzy_name_difference = fields.Integer(
+        string="Max. characters difference", default=2)
```

Comment:
Have you considered making this a percent value? Maybe I'm saying an stupidity, but if a name is 5 characters long and 2 differ, seems equivalent to 20 characters in a 100-char-long name.

Setting it to 2% (rounded up) would make it more consistent IMHO.